js/samples/quickstart-kotlin/app/src/main/java/com/example/immersivereadersdk/MainActivity.kt

```diff
@@ -94,15 +94,23 @@ class MainActivity : AppCompatActivity() {
         var token: String
 
         runBlocking{
-            val resp = async { getImmersiveReaderTokenAsync() }
+            val resp = async { getImmersiveReaderTokenAsync(tokenFromServer) }
             token = resp.await()
-            val jsonResp = JSONObject(token)
-            loadImmersiveReaderWebView(exampleActivity, jsonResp.getString("access_token"), subdomain, content, options)
+            if (!tokenFromServer) {
```

Comment:
code to get token from server should be in an else block

Reply:
done